compose/src/main/kotlin/io/github/kakaocup/compose/rule/KakaoComposeTestRule.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.kakaocup.compose.rule
+
+import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
+import io.github.kakaocup.compose.KakaoCompose
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class KakaoComposeTestRule(
```

Comment:
Don't you want to add some additional factory functions for simple cases out of the box? 🤔 

Something like:

```kotlin
object KakaoTestRulesFactory {
     
    fun createEmptyComposeRule(): KakaoComposeTestRule {
          val composeTestRule = createEmptyComposeRule()
          return KakaoComposeTestRule(composeTestRule)
     }

     inline fun <reified A : ComponentActivity> createAndroidComposeRule<A>(): KakaoComposeTestRule {
        val composeTestRule = createAndroidComposeRule<A>()
        return KakaoComposeTestRule(composeTestRule)
     }

     // ...
}

```

Reply:
That implementation will not work because:
1. `val composeTestRule = createEmptyComposeRule()` will not apply `environment` as `@Rule`. 
2. Creating new instance is not the same if using defined `@Rule` one.